### DISCUSSION TITLE
Audio coding stuck at 32kHz

### DISCUSSION THREAD

**Discussion by mdehrlich** [GerryDazoo/Slinger#264] [General]
For the first time I tried using 48 and 64.  Regardless of the setting, VLC tells me that the audio codec is 32kHz.  The log file shows that slinger is sending the correct value to the slingbox.  If we have to stick with 32, it's ok.

**Comment by GerryDazoo** [comment]
Not really.  Here's the dump from my media player that says it's seeing a 64kbs audio stream.

  Stream #0:0: Video: h264 (h264 / 0x34363268), none, 1280x720, 1k tbr, 1k tbn
  Stream #0:1: Audio: aac (LC) ([255][0][0][0] / 0x00FF), 32000 Hz, stereo, fltp, **64 kb/s**

I can change the audio bit rate in Slinger and these numbers change.
  Stream #0:1: Audio: aac (LC) ([255][0][0][0] / 0x00FF), 32000 Hz, stereo, fltp, **96 kb/s**

I think the 32000Hz is coming from the metadata coming from the slingbox. But the sling box is sending the correct bit rate.

**Comment by mdehrlich** [comment]
What media player do you use? VLC only shows 32kHz

**Reply by GerryDazoo** [reply]
SMPlayer and ffPlayer.  Both those show the expected audio bit rate I gave up on VLC awhile ago.

**Comment by mdehrlich** [comment]
Ok, I installed SMPlayer on my laptop and android phone.   32 and 64 produce the same bitrate of about 64.  96 does give me 96.  All have a 32kHz sample rate, so I don't know what benefit 96 produces.
